src/Microsoft.Health.Fhir.Core/Features/Operations/Import/ImportOrchestratorJob.cs

```diff
@@ -277,35 +279,121 @@ private async Task SendImportMetricsNotification(JobStatus jobStatus, JobInfo jo
             await _mediator.Publish(importJobMetricsNotification, CancellationToken.None);
         }
 
-        private async Task ExecuteImportProcessingJobAsync(IProgress<string> progress, JobInfo jobInfo, ImportOrchestratorJobInputData inputData, ImportOrchestratorJobResult currentResult, CancellationToken cancellationToken)
+        private async Task ExecuteImportProcessingJobAsync(IProgress<string> progress, JobInfo coord, ImportOrchestratorJobInputData coordDefinition, ImportOrchestratorJobResult currentResult, bool isMerge, CancellationToken cancellationToken)
```

Comment:
In merge import all child jobs are created by one enqueue call in single SQL transaction. If restart of coordinator happens before that - nothing registered. If after, then enqueue is idempotent, so it will return the same previously registered jobs.